source/features/home-hotkey.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'dom-chef';
+import onetime from 'onetime';
+import {isEnterprise} from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const homepageLink = (isEnterprise() ? location.origin : 'https://github.com');
```

Comment:
```suggestion
	const homepageLink = isEnterprise() ? location.origin : 'https://github.com';
```

Reply:
They are identical, there should not a be a condition at all. It's always `location.origin` and it should just be inlined in the JSX.